boilerplate/app/services/reactotron/reactotron-client.web.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This file is loaded in web and exports the React.js version
+ * of Reactotron's client.
+ *
+ * React Native is loaded from reactotron-client.ts.
+ */
+import Reactotron from "reactotron-react-js"
+export { Reactotron }
```

Comment:
These are just renamed from `tron.web.ts` and `tron.ts`, which I didn't think was very good.